rpms/pulp-scl/pulp-scl.spec

```diff
@@ -0,0 +1,153 @@
+# We specificall separate the 'pulp' and '-scl' parts from each other.
+# since spec file macros don't allow '-' and the auto generated ones
+# are based on %scl_name
+%global scl_name_base pulp
+%global scl_name_postfix -scl
+
+%global scl_vendor pulp
+%global _scl_prefix /opt/%{scl_vendor}
+
+# scl specific macros
+%global scl %{scl_name_base}%{scl_name_postfix}
+%scl_package %scl
+
+# Python scl macros
+%{!?scl_python:%global scl_python rh-python35}
+%{!?scl_prefix_python:%global scl_prefix_python %{scl_python}-}
+
+# Only for this build, you need to override default __os_install_post,
+# because the default one would find /opt/.../lib/python2.7/ and try
+# to bytecompile with the system /usr/bin/python2.7
+%global __os_install_post %{%{scl_no_vendor}_os_install_post}
+# Similarly, override __python_requires for automatic dependency generator
+%global __python_requires %{%{scl_no_vendor}_python_requires}
+
+# Generate our sitelib variable from %scl_python's sitelib variable.  
+# Secifically, we replace /rh/%{scl_python} with /pulp/pulp-scl/
+%global pulp_sitelib %(echo %{python35python_sitelib} | sed 's|/rh/%{scl_python}|/%{scl_vendor}/%{scl}|')
+
+%global debug_package %{nil}
+%global nfsmountable 1
+
+Name:		%scl_name
+Version:	0.1
+Release:	12%{?dist}
+Summary:	Package that installs %scl
+
+License:	GPLv2+
+Source0:	README
+Source1:	LICENSE
+
+# Basic scl dependencies
+BuildRequires:	scl-utils-build
+BuildRequires:	help2man
+
+# Python scl dependencies
+BuildRequires:	%{scl_prefix_python}scldevel
+BuildRequires:	%{scl_prefix_python}runtime
+BuildRequires:	%{scl_prefix_python}build
+BuildRequires:	%{scl_prefix_python}python-devel
+
+Requires:	%{scl_python}
+
+%description
+This is the main package for %scl Software Collection
+
+Provides dependencies for pulp (http://pulpproject.org)
+
+%package runtime
+Summary: Package that handles %scl Software Collection
+
+Requires: scl-utils
+Requires: %{scl_prefix_python}runtime
+
+%description runtime
+Package shipping essential scripts to work with %scl Software Collection.
+
+Provides dependencies for pulp (http://pulpproject.org)
+
+%package build
+Summary: Package shipping basic build configuration
+Requires: scl-utils-build
+Requires: %{scl_prefix_python}scldevel
+
+%description build
+Package shipping essential configuration macros to build %scl Software Collection.
+
+Provides dependencies for pulp (http://pulpproject.org)
+
+%package scldevel
+Summary: Package shipping development files for %scl
+
```

Comment:
according to https://www.softwarecollections.org/en/docs/guide/#sect-Providing_an_scldevel_Subpackage 
it might be best practice to have a ​line for `Provides: scldevel(%{scl_name_base})`